### PR TITLE
Added cos math method in java concepts

### PR DESCRIPTION
**Pull request by ShreyasPatil-13**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/math-methods/terms/cos/cos.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.cos()`** method returns the trigonometric cosine of an angle argument, given in radians.
```
Comment (dakshdeepHERE) [inline content/java/concepts/math-methods/terms/cos/cos.md:16]: Be a bit more descriptive here please.

File: content/java/concepts/math-methods/terms/cos/cos.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+## Syntax
+
+```pseudo
+Math.cos(angle)
+```
+
+The `angle` parameter is expressed in radians.
+
+- The result is in the range [-1, 1].
+- If the `angle` is `NaN` or infinity, `NaN` is returned.
```
Comment (dakshdeepHERE) [inline content/java/concepts/math-methods/terms/cos/cos.md:27]:
```suggestion
- The result is in the range [-1, 1].
- If the `angle` is **NaN** or **Infinity**, **NaN** is returned.

> **Note:** The `angle` parameter is expressed in radians.
```

File: content/java/concepts/math-methods/terms/cos/cos.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Example
+
+The following example demonstrates the application of `.cos()` method:
+
+```java
+// Check.java
```
Comment (dakshdeepHERE) [inline content/java/concepts/math-methods/terms/cos/cos.md:34]:
```suggestion
```

File: content/java/concepts/math-methods/terms/cos/cos.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    // convert degrees to radians
+    double pi = Math.PI;
+    double degree = 60;
+    double radian = degree * pi/180;
+    System.out.println( "Cosine of 60 degrees is " + Math.cos(radian));
+  }
+}
+```
+
+This results in the following output:
```
Comment (dakshdeepHERE) [inline content/java/concepts/math-methods/terms/cos/cos.md:46]:
```suggestion
The above example results in the following output:
```

File: content/java/concepts/math-methods/terms/cos/cos.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+---
+Title: '.cos()'
+Description: 'Returns the trigonometric cosine of the specified angle.'
+Subjects:
+  - 'Computer Science'
```
Comment (dakshdeepHERE) [inline content/java/concepts/math-methods/terms/cos/cos.md:5]:
```suggestion
  - 'Computer Science'
  - 'Code Foundations'
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I have some comment for you to work on @ShreyasPatil-13.

**Comment by dakshdeepHERE** [comment]
Is there any update on this PR @ShreyasPatil-13 ? Please work on it or we will have to un assign you from the issue.

**Review by dakshdeepHERE — APPROVED** [review]
LGTM 🚀

File: content/java/concepts/math-methods/terms/cos/cos.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    double radian = degree * pi/180;
+    System.out.println( "Cosine of 60 degrees is " + Math.cos(radian));
+  }
+}
+```
+
+The above example results in the following output:
+
+```shell
+Cosine of 60 degrees is 0.5
```
Comment (ishg-153) [inline content/java/concepts/math-methods/terms/cos/cos.md:49]: When I ran the above code block, I received the following output:
`Cosine of 60 degrees is 0.5000000000000001`
Can you check and confirm your output? It is expected to be exact.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@ShreyasPatil-13 I have suggested some changes. Please make them and ping me once done.

**Comment by ishg-153** [comment]
Hey @ShreyasPatil-13! I see you've applied two of my suggested changes. Can you please re run the example code block and enter the exact output in ` ```shell ``` `.

File: content/java/concepts/math-methods/terms/cos/cos.md (hunk 6 above)
Comment (dakshdeepHERE) [reply content/java/concepts/math-methods/terms/cos/cos.md:48]: @ishg-153 check the code output now

**Review by ishg-153 — APPROVED** [review]
Merging

**Pull request merged**